### ISSUE TITLE
How to specify the timeline in which we want to extract the posts?

### ISSUE THREAD

**Issue by Nitigya-Handa** [kevinzg/facebook-scraper#100]
Is there a way I can specify dates both from and upto which I want to extract the posts from?

**Comment by kevinzg** [comment]
No, the scraper just open the FB page at the latest posts and then "scrolls down" to get more posts.
I don't know if there is a way to request only posts from a certain range.

**Comment by Nitigya-Handa** [comment]
There must be a way because I had used a Twitter Scraper GetOldTweets3 which has a library known as TweetCriteria using which one was able to set the timeline and define keywords or hashtags to search/extract the tweets with.

**Comment by widedM** [comment]
You can create a list of keywords and check if the post have minimum one word of that list. It solve the problem

**Comment by GivenToFlyCoder** [comment]
Use date time variable "time" as "from date limit",  implement it in a while loop that breaks getting post if the date publication is less than you desired. 

Upto date limit i don´t think it can be possible because it scrolls down from actual date extractio, but you can store it and then filter it.

**Comment by kevinzg** [comment]
Note that some pages have sticky posts, so it's a good idea skip the check for the first posts of the page.
